src/components/import_from_slurp/import_from_slurp_form.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react'
```

Comment:
The import_from_slurp component should live within the space, probably the space show.